cropharvest/eo/eo.py

```diff
@@ -38,6 +38,29 @@
 STATIC_IMAGE_FUNCTIONS = [get_single_srtm_image]
 
 
+def memoize(f):
```

Comment:
Can you use the memoization defined [here](https://github.com/nasaharvest/cropharvest/blob/main/cropharvest/utils.py#L112)?

Reply:
Good catch